run_integration.py

```diff
@@ -0,0 +1,13 @@
+import os
+import sys
+
+if __name__ == "__main__":
+    if sys.argv[1] and sys.argv[2]:
+        os.environ["C42_USER"] = sys.argv[1]
+        os.environ["C42_PW"] = sys.argv[2]
+        rc = os.system("pytest ./integration -v -rsxX -l --tb=short --strict")
+        sys.exit(rc)
+    else:
```

Comment:
This should fail, so you can raise an exception like `raise Exception("Error: missing Code42 username and password parameters")`

Reply:
Changed.